src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar_mon">
```

Comment:
You still haven't added modifiers for the calendar, there should be a BEM modifier for what day the calendar starts on and how many days are in the month

Reply:
added those, but had to add calender class wrapper as well, cause test were screaming on me for parent block
please see 'v4'